File: flask-backend/models/product.py
from extensions import db


class Product(db.Model):

    __tablename__ = "products"

    productId = db.Column(db.Integer, primary_key=True)
    productName = db.Column(db.String(255), nullable=False)
    description = db.Column(db.String(255), nullable=False)
    price = db.Column(db.Float, nullable=False)
    category = db.Column(db.String(255), nullable=False)
    image = db.Column(db.String(255), nullable=True)

File: spring-boot-backend/src/main/java/com/springbootbackend/model/Product.java
package com.springbootbackend.model;


import jakarta.persistence.*;
import lombok.*;

@Data
@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name= "products")
public class Product {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long productId;

    private String productName;
    private String description;
    private Double price;
    private String category;
}
